src/components/data/row/IconDataRow.tsx

```diff
@@ -31,7 +31,7 @@ const IconDataRowComponent = (props: Props) => {
       <View style={styles.leftColumn}>
```

Comment:
The row-based layout discussed actually won't work because another caller of IconDataRow (PaymentMethodRow) is reliant on the two column base layout style. Changed it so that the top left and top right texts are fixed in size. This way also we retain the left justified ticker.